uber/automated_emails.py

```diff
@@ -14,14 +14,16 @@ class AutomatedEmail:
     }
     extra_models = queries  # we've renamed "extra_models" to "queries" but are temporarily keeping the old name for backwards-compatibility
 
-    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
+    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True, email_id=None):
```

Comment:
``email_id`` is a fine enough name for this.  However, it might be slightly confusing in that every other place where we have "id" in our codebase, we're referring to a guid like ``ca917be9-41a3-4c3e-a683-32b2c0cb2a75``.

What we're talking about is a unique textual, human-generated, human-readable identifier.  The term I've seen used for that is ``ident``, so that might be a good name for this.  Though really ``email_id`` is fine since it's not inaccurate or misleading in any way that I can see mattering.